docs/source/tutorials/instrument-modeling/equity-extension.rst

```diff
@@ -145,13 +145,22 @@ components:
 - An option instrument, which describes the economic term of the rights a shareholder recieves.
   For example, this could be a European option with a strike price below the current spot price, and
   a maturity three weeks in the future.
-  The :doc:`Generic Tutorial <generic-extension>` describes how to create and lifecycle a European
-  option.
+  The :doc:`Generic Tutorial <generic-extension>` describes how to create a European option.
 - The ``DeclareDividend`` choice to distribute the above option instrument in the correct proportion
   (e.g. 3 option contracts for each 10 shares held). This can be done in the same way as the Bonus
   Issue example described earlier, just change the ``perUnitDistribution`` line to distribute the
   option instrument you created above.
 
+When current shareholders receive the option instrument they can typically choose between:
+
+#. Exercising the option. The :doc:`Generic Tutorial <generic-extension>` describes how to elect to
+   exercise the option.
+#. Doing nothing. The option will expire worthless.
```

Comment:
that's not exactly true, the `or` combinator in contingent claims forces you to choose explicitly between `exercise` and `do not exercise`

Reply:
Thanks! I have clarified this now to indicate an active choice.